### ISSUE TITLE
rdpmc instruction causes xhyve abort for boot2docker guest

### ISSUE THREAD

**Issue by romankulikov** [mist64/xhyve#69]
Simplest test program:
```
int main()
{
        asm("rdpmc");
        return 0;
}
```

Being run, for example, in Ubuntu 14.04 Server crashes as expected. But in Boot2Docker 1.7 guest xhyve aborts with VMX vmexit reason: https://github.com/ailispaw/boot2docker-xhyve/issues/6

**Comment by xez** [comment]
Yeah, `rdpmc` is not enabled for the guest. Probably shouldn't be advertising that instruction in `cpuid`.

**Comment by ailispaw** [comment]
It looks the performance feature has already been disabled.
https://github.com/mist64/xhyve/blob/master/src/vmm/x86.c#L253
https://github.com/mist64/xhyve/blob/master/src/vmm/x86.c#L343-L352

In addition, I put the following in https://github.com/mist64/xhyve/blob/master/src/vmm/intel/vmx.c#L1850.
```
	case EXIT_REASON_RDPMC:
		// RDPMC is not supported.
		vm_inject_gp(vmx->vm, vcpu);
		handled = HANDLED;
		break;
```
It doesn't look nice, but It works fine for me so far.

**Comment by ailispaw** [comment]
And also the @romankulikov's test program will get `Segmentation fault` with my patch.
As expected?

**Comment by romankulikov** [comment]
@ailispaw yes, SIGSEGV is an expected result for the test program. And RDPMC handler suggested above seems will fix the problem with xhyve abort on Parallels Tools startup check.

**Comment by ailispaw** [comment]
FYI:
I don't know why, but it seems to work fine with boot2docker v1.8.3 and xhyve v0.2.0 now.
https://github.com/ailispaw/boot2docker-xhyve/commit/52d75132f361ca21a3e6f9bdb4baff607b7cbf49

**Comment by xez** [comment]
I guess we should report the number of available counters as 0 in `cpuid` leaf 0xa. That way Linux shouldn't be using, or allowing user space to use, `rdpmc` in the first place.

**Comment by ailispaw** [comment]
@xez 
https://github.com/mist64/xhyve/blob/master/src/vmm/x86.c#L343-L352
How about this?
